src/main/java/com/znsio/e2e/tools/Driver.java

```diff
@@ -420,4 +421,23 @@ public void scrollTillElementIntoView(By elementId) {
         WebElement element = driver.findElement(elementId);
         ((JavascriptExecutor) driver).executeScript("arguments[0].scrollIntoView(true);", element);
     }
+    public void switchToNextTab() {
+        Iterator<String> iterator = driver.getWindowHandles().iterator();
+        try {
+            iterator.next();
+            driver.switchTo().window(iterator.next());
+        } catch (NoSuchElementException e) {
+            LOGGER.error("Unable to get next window handle.");
+            e.printStackTrace();
```

Comment:
we should not swallow the exception. There should be an exception thrown here 
ex:

`throw new NoSuchElementException("Unable to get next window handle.", e);`

In such approch, `e.printStackTrace();` is also not required

Reply:
updated